protocols/request-response/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-request-response"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Generic Request/Response Protocols"
-version = "0.26.1"
+version = "0.27.0"
```

Comment:
Same here, this is backwards compatible so we can just bump this to `0.26.2`.

Reply:
:heavy_check_mark: